Automattic/jetpack
# Connection flows: code cleanup / removal of deprecated tools

Fixes #35603

## Proposed changes:

Follow-up to #34768

Now that we do not have connection screens outside of the Jetpack plugin screens, we do not need to keep the different resources removed in this commit.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

You will need to test this in different environments:

- On single sites
- On multisites

* Start with sitse that are not connected to WordPress.com yet, and where the plugin is not active.
* Activate the plugin
* Go to Jetpack > Dashboard and Jetpack > My Jetpack
    * Test that the connection flows are complete with nothing missing.
* Post-connection, check that the Recommendations screen under Jetpack > Dashboard > Recommendations still works.
* Go to the Plugins menu and try deactivating the plugin; a deactivation modal should come up, where everything works.
